mobile/src/main/res/layout/list_item_menu_item.xml

```diff
@@ -24,9 +26,20 @@
         android:id="@+id/price"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
-        android:layout_alignParentLeft="true"
-        android:layout_alignParentStart="true"
-        android:layout_below="@id/name"
-        android:textAppearance="?android:attr/textAppearanceSmall"
+        android:layout_alignParentEnd="true"
+        android:layout_alignParentRight="true"
+        android:layout_alignParentTop="true"
+        android:textAppearance="?android:attr/textAppearanceMedium"
+        android:textStyle="bold"
         tools:text="$4.25" />
+
+    <TextView
+        android:id="@+id/description"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_below="@+id/name"
+        android:layout_marginTop="8dp"
+        android:textAppearance="?android:attr/textAppearanceSmall"
+        android:textStyle="italic"
+        tools:text="Pulled Rotisserie Chicken, Blue Cheese Spread, Organic Mixed Greens, Bernie's Bacon, Avocado, Oven Roasted Tomatoes Topped With Truck-Made Hot Sauce" />
```

Comment:
I'm guessing this was for testing how the layout looked. Does it still need to be here?